sagemaker-fundamentals/create-training-job/create-training-job.ipynb

```diff
@@ -0,0 +1,856 @@
+    "    \"Statement\": [\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "                \"s3:Get*\",\n",
+    "                \"s3:Put*\"\n",
+    "            ],\n",
+    "            \"Resource\": bucket_arn\n",
+    "        }\n",
+    "    ]\n",
+    "}\n",
+    "\n",
+    "pm_res = attach_permission(\n",
+    "    role_name=role['RoleName'],\n",
+    "    policy_name='get_put',\n",
+    "    policy_doc=get_put)\n",
+    "\n",
+    "pp.pprint(pm_res)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_prefix = 'input_data/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# copy from sagemaker-samplef-files to {bucket}\n",
+    "s3 = boto3.client('s3')\n",
+    "\n",
+    "# copy remote csv files to local\n",
+    "files = []\n",
+    "data_dir = '/tmp'\n",
+    "for obj in obj_res['Contents']:\n",
+    "    if '/tabular/fraud_detection/synthethic_fraud_detection_SA' in obj['Key']:\n",
+    "        key = obj['Key']\n",
+    "        if key.endswith('.csv'):\n",
+    "            filename=key.split('/')[-1]\n",
+    "            files.append(filename)\n",
+    "            with open(os.path.join(data_dir, filename), 'wb') as f:\n",
+    "                s3.download_fileobj(public_bucket, key, f)\n",
+    "\n",
+    "# upload from local to the bucket you just created\n",
+    "for fname in files:\n",
+    "    with open(os.path.join(data_dir, fname), 'rb') as f:\n",
+    "        key = input_prefix + fname\n",
+    "        s3.upload_fileobj(f, bucket, key)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# inspect your bucket\n",
+    "obj_res = s3.list_objects_v2(\n",
+    "    Bucket=bucket)\n",
+    "\n",
+    "for obj in obj_res['Contents']:\n",
+    "    print(obj['Key'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare an S3 URI for saving model artifact\n",
+    "\n",
+    "After your image is done with model training, it needs to write the trained model artifact into `/opt/ml/model`. This is directory where SageMaker looks for the trained model artifact and upload it to an S3 URI you will configure later. Naturally, the execution role `sm` needs to have write permission to this S3 URI. \n",
+    "\n",
+    "Refer to the section on [How Amazon SageMaker Processes Training Output](https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo-output.html) in the official docs for more detail. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Put everything together\n",
+    "\n",
+    "Now, you have everything you need to create a training job. Let's review what you have done. you have \n",
```

Comment:
Capitalize "You..."

Reply:
Like "Now You have everything ...." ? what's the idea?

Reply:
Check the third sentence: "...Let's review what you have done. you have...." <- capitalize this one.